hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeStateMachine.java

```diff
@@ -260,7 +261,7 @@ public DatanodeStateMachine(DatanodeDetails datanodeDetails,
 
   @VisibleForTesting
   public DatanodeStateMachine(DatanodeDetails datanodeDetails,
-                              ConfigurationSource conf) throws IOException {
+                              OzoneConfiguration conf) throws IOException {
```

Comment:
Why we change the conf from ConfigurationSource to OzoneConfiguration? it's also changed in below OzoneContainer.java.

Reply:
@xichen01 , is this one missed?

Reply:
Done